content/legal/release-policy.md

```diff
@@ -79,6 +79,11 @@ earlier, reporting can be done in the project's next scheduled board report.
 
 Projects SHALL publish official releases and SHALL NOT publish unreleased
 materials outside the development community.
+Projects SHALL provide one or more Project Download Pages as part of their web site.
```

Comment:
Need to check with Infra here - AIUI, there has been some talk of standardising download pages; these would not be part of the project website.

Also, emphasising PROJECT might be confusing for podlings. I'm not sure it is necessary to coin a new phrase; they are just download pages

Reply:
@clambertus There are no changes proposed to the release policy. 
This is a clarification because of projects' confusion about what belongs in announcements.